### ISSUE TITLE
RFC: "debug" prop for useSpring et al

### ISSUE THREAD

**Issue by aleclarson** [react-spring/react-spring#666]
`debug` prop for `useSpring` et al which:
- enables more logs
- does other stuff? records in chrome devtools?

**Comment by aleclarson** [comment]
Slightly related: Maybe we should add a `dilation` property to the `config` prop, which "expands" the frame rate when below 1 and "shrinks" the frame rate when above 1. For example, a dilation of 2 would make the animation twice as fast. The main use case is a slow-motion effect for debugging purposes.

**Comment by fandy** [comment]
Yeah this would be super useful. When an animation is performing badly it's really hard to debug without Chrome Dev Tools.
